week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,19 +19,23 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
-
+function validate(num) {
+  if (typeof num === "number" && num % 2 === 0 && num<= 100 ){
```

Comment:
You can actually just write 

`return typeof num === "number" && num % 2 === 0 && num<= 100`

because that statement will `return` true or false anything. Give it a try!